### PR TITLE
Domaintools Enterprise pack v1.1.6

### PR DESCRIPTION
**Pull request by briluza**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Change test endpoint and added app_partner, app_name, & app_version in every API request for better reporting

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/DomainTools/ReleaseNotes/1_1_6.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+
+##### DomainTools
+- Change test endpoint to /v1/account/
```
Comment (Shellyber) [inline Packs/DomainTools/ReleaseNotes/1_1_6.md:4]:
```suggestion
```

File: Packs/DomainTools/ReleaseNotes/1_1_6.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+- Add app_partner, app_name, & app_version in API requests for better reporting.
```
Comment (Shellyber) [inline Packs/DomainTools/ReleaseNotes/1_1_6.md:5]:
```suggestion
- Added the following parameters to the API requests for better reporting:
  - *app_partner*
  - *app_name*
  - *app_version*
```

**Review by Shellyber — COMMENTED** [review]
Thanks for your contribution. Looks good.
Small note regarding the release note

File: Packs/DomainTools/ReleaseNotes/1_1_6.md (hunk 1 above)
Comment (briluza) [reply Packs/DomainTools/ReleaseNotes/1_1_6.md]: done

File: Packs/DomainTools/ReleaseNotes/1_1_6.md (hunk 2 above)
Comment (briluza) [reply Packs/DomainTools/ReleaseNotes/1_1_6.md]: done

**Review by Shellyber — APPROVED** [review]
nice!

**Pull request merged**